### ISSUE TITLE
Issue with PsTestTool-4 

### ISSUE THREAD

**Issue by gintuoklis** [microsoft/navcontainerhelper#700]
Hey @freddydk, 

I am using your CI pipelines scripts from for my CI pipelines (bc 14)
https://dev.azure.com/businesscentralapps/HelloWorld/_build?definitionId=4

I using a setting to reuse the container to speed up the process. It works fine in the first run, but on the next run it fails on the **Run Tests** phase. 

See the error bellow:

```
ERROR: Cannot build the page 130409.
The metadata object Page 130409 was not found.
Cannot open page 130409. You might need to import the test toolkit to the container and/or remove the folder C:\ProgramData\NavContainerHelper\Extensions\Agent-build\PsTestTool-4 and retry.
at Get-Tests, C:\ProgramData\NavContainerHelper\Extensions\Agent-build\PsTestTool-4\PsTestFunctions.ps1: line 172
```

If I login to the buildagent and manually remove the PsTestTool-4 folder, it works fine for once run again. And then it fails. 

Any ideas why it's happening? Do you have any suggestions how to solve it?

**Comment by freddydk** [comment]
Yes, I know why this is happening.
It is a bug

**Comment by freddydk** [comment]
The bug is in 14.x and before.
Solution is probably to delete the folder when restoring a database, will check that

**Comment by freddydk** [comment]
Shipped in 0.6.4.17
